view_presentation/README.md

```diff
@@ -0,0 +1,3 @@
+## View Presentation
```

Comment:
i understand this README will contain information for the user of the view presentation module.

I imagine it'll also include a link to the information necessary to add other interfaces and why one may want to do such thing, in interface/README

Reply:
Yes, the main view-presentation.py is under construction and will be pushed soon. Yes, this readme will link with the interface/readme.